docs/core_systems/service_registry.md

```diff
@@ -173,7 +173,105 @@ Auto initialization of database tables. Value must be always 'none'.
 
 ### Custom parameters
 
-TODO
+:fontawesome-solid-wrench: **authentication.policy**
+
+Way of authenticating the service requester systems. Can be `declared`, `certificate` or `outsourced`.
```

Comment:
In my branch (auth idd), I moved this to the general parameters. Every system we are implementing or will implement use this.